use-timescale/ingest-data/import-parquet.md

```diff
@@ -1,73 +1,105 @@
+
+# Import data from Parquet
+
+[Apache Parquet][apache-parquet] is a free and open-source column-oriented data storage format in the 
+Apache Hadoop ecosystem. It provides efficient data compression and encoding schemes with 
+enhanced performance to handle complex data in bulk. 
+
+This page shows you how to import data into your $SERVICE_LONG from a Parquet file.
 
 ## Prerequisites
 
-Before you start, make sure you have:
+Add ImportPrerequisites here 
+- [Install DuckDB][install-duckdb] on the source machine where the Parquet file is located.
+- Ensured that the time column in the Parquet file uses the `TIMESTAMP` data type. 
 
-- Checked that the parquet file schema is compatible with the table you want to import it into.
-- Ensured that the time  column in the source data uses the TIMESTAMPTZ  data type.
-- Install DuckDB on the source machine where the Parquet file is located. You can download 
-  and install DuckDB from https://duckdb.org/docs/installation/
+## Import data into your $SERVICE_SHORT
 
-## Import data
+To import data from a Parquet file:
 
-Follow these steps to import data from a Parquet file:
+<Procedure>
 
-1.  Create a new empty table: Use a schema that matches the data in your parquet file. In this example, the parquet file contains the columns ts, location, and temperature with types 
-    TIMESTAMP, STRING, and DOUBLE.
+1. **Setup your connection string**
 
-    ```sql
-    CREATE TABLE <TABLE_NAME> (
-        ts          TIMESTAMPTZ         NOT NULL,
-        location    TEXT                NOT NULL,
-        temperature DOUBLE PRECISION    NULL
-    );
-    ```
+    <SetupConnectionString />
 
-2.  Convert the empty table to a hypertable: Use the create_hypertable function. 
-    Replace <TABLE_NAME> and <COLUMN_NAME> with the name of the column storing time values in your table.
+1. **Create a [hypertable][hypertable-docs] to hold your data** 
 
-    ```sql
-    SELECT create_hypertable('<TABLE_NAME>', by_range('<COLUMN_NAME>'))
-    ```
+   1.  Create a new empty table with a schema that is compatible with the data in your parquet file.  
+  
+       For example, if your parquet file contains the columns `ts`, `location`, and `temperature` with types 
+       `TIMESTAMP`, `STRING`, and `DOUBLE`:
+  
+       ```sql
+       psql $TARGET -c  "CREATE TABLE <TABLE_NAME> ( \
+           ts          TIMESTAMPTZ         NOT NULL,  \
+           location    TEXT                NOT NULL,  \
+           temperature DOUBLE PRECISION    NULL  \
+       );"
+       ```
+       If you prefer using a secure UI to the command line, use [Data mode in $CONSOLE][data-mode].
 
-3.  Start a new DuckDB interactive session. In your system's command line interface, run:
+   1.  Convert the empty table to a hypertable:
 
-    ```bash
-    duckdb
-    ```
+       Replace the `<META NAMES>` with the correct column names, then create the hypertable.
```

Comment:
The <META NAMES> sounds confusing its only one column name how about we add like
where <TABLE_NAME> is the table created in the previous step
<COLUMN_NAME> is the partitioning column that is timestamp column.